javascriptv3/example_code/sagemaker/scenarios/wkflw-sagemaker-geospatial-pipeline/SageMakerPipelinesWkflw.js

```diff
@@ -0,0 +1,387 @@
+    const {
+      arn: pipelineExecutionRoleArn,
+      cleanUp: pipelineExecutionRoleCleanUp,
+    } = await createSagemakerRole({
+      iamClient: this.clients.IAM,
+      name: this.names.SAGE_MAKER_EXECUTION_ROLE,
+    });
+    this.cleanUpFunctions.push(pipelineExecutionRoleCleanUp);
+
+    await this.logger.log(
+      MESSAGES.roleCreated.replace(
+        "${ROLE_NAME}",
+        this.names.SAGE_MAKER_EXECUTION_ROLE
+      )
+    );
+
+    this.prompter.logSeparator();
+
+    // Create an IAM policy that allows the AWS Lambda function to invoke SageMaker APIs.
+    const {
+      arn: lambdaExecutionPolicyArn,
+      policy: lambdaPolicy,
+      cleanUp: lambdaExecutionPolicyCleanUp,
+    } = await createLambdaExecutionPolicy({
+      name: this.names.LAMBDA_EXECUTION_ROLE_POLICY,
+      s3BucketName: this.names.S3_BUCKET,
+      iamClient: this.clients.IAM,
+      pipelineExecutionRoleArn,
+    });
+    this.cleanUpFunctions.push(lambdaExecutionPolicyCleanUp);
+
+    console.log(JSON.stringify(lambdaPolicy, null, 2), "\n");
+
+    await this.logger.log(
+      MESSAGES.attachPolicy
+        .replace("${POLICY_NAME}", this.names.LAMBDA_EXECUTION_ROLE_POLICY)
+        .replace("${ROLE_NAME}", this.names.LAMBDA_EXECUTION_ROLE)
+    );
+
+    await this.prompter.checkContinue();
+
+    // Attach the Lambda execution policy to the execution role.
+    const { cleanUp: lambdaExecutionRolePolicyCleanUp } = await attachPolicy({
+      roleName: this.names.LAMBDA_EXECUTION_ROLE,
+      policyArn: lambdaExecutionPolicyArn,
+      iamClient: this.clients.IAM,
+    });
+    this.cleanUpFunctions.push(lambdaExecutionRolePolicyCleanUp);
+
+    await this.logger.log(MESSAGES.policyAttached);
+
+    this.prompter.logSeparator();
+
+    // Create Lambda layer for SageMaker packages.
+    const { versionArn: layerVersionArn, cleanUp: lambdaLayerCleanUp } =
+      await createLambdaLayer({
+        name: this.names.LAMBDA_LAYER,
+        lambdaClient: this.clients.Lambda,
+      });
+    this.cleanUpFunctions.push(lambdaLayerCleanUp);
+
+    await this.logger.log(
+      MESSAGES.creatingFunction.replace(
+        "${FUNCTION_NAME}",
+        this.names.LAMBDA_FUNCTION
+      )
+    );
+
+    // Create the Lambda function with the execution role.
+    const { arn: lambdaArn, cleanUp: lambdaCleanUp } =
+      await createLambdaFunction({
+        roleArn: lambdaExecutionRoleArn,
+        lambdaClient: this.clients.Lambda,
+        name: this.names.LAMBDA_FUNCTION,
+        layerVersionArn,
+      });
+    this.cleanUpFunctions.push(lambdaCleanUp);
+
+    await this.logger.log(
+      MESSAGES.functionCreated.replace(
+        "${FUNCTION_NAME}",
+        this.names.LAMBDA_FUNCTION
+      )
+    );
+
+    this.prompter.logSeparator();
+
+    await this.logger.log(
+      MESSAGES.creatingSQSQueue.replace("${QUEUE_NAME}", this.names.SQS_QUEUE)
+    );
+
+    // Create SQS queue for the SageMaker pipeline.
```

Comment:
Create "the", or create "an" SQS queue, maybe?

Reply:
Sounds better. Fixed.